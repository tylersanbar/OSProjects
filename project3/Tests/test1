#!/bin/bash
# Format test
./myfs_format 64
./myfs_inspect -volume
./myfs_inspect -iblock 1
./myfs_inspect -dir 2
