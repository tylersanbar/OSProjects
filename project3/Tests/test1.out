N_BLOCKS: 64
N_ALLOCATED_BLOCKS: 3
N_ALLOCATED_INDEX_NODES: 1
Block allocation table:
00: 07
01: 00
02: 00
03: 00
04: 00
05: 00
06: 00
07: 00
Relative Index Node 0	DIRECTORY	Nref=1	Content=2	size=2
Next block: 65534
Directory at block 2:
Entry 0: name=".", index_node=0
Entry 1: name="..", index_node=0
Next block: 65534
